test/diff.jl

```diff
@@ -318,3 +318,10 @@ let
     @test !iszero(sj)
     @test isequal(sj, Symbolics.jacobian(Symbolics.scalarize.(y), x))
 end
+
+# substituting iv of differentials
+@variables t t2 x(t)
+D = Differential(t)
+ex = D(x)
+ex2 = substitute(ex, [t=>t2])
```

Comment:
Could you test `substitute(D(x) * 2 + x / t, [t=>t2])`? We need to make sure that it recuses correctly. Sub one expression is too simple.